src/layouts/ResourceCategory/ResourceCategory.tsx

```diff
@@ -51,22 +61,44 @@ export const ResourceCategory = (): JSX.Element => {
           </Box>
         </Section>
         <Section>
-          <Box w="full">
-            <SectionHeader label="Resource Pages">
-              <CreateButton as={RouterLink} to={`${url}/createPage`}>
-                Create page
-              </CreateButton>
-            </SectionHeader>
-            <SectionCaption icon={BiBulb} label="NOTE: ">
-              Pages are automatically ordered by latest date.
-            </SectionCaption>
-          </Box>
+          {!isPagesEmpty && (
+            <Box w="full">
+              <SectionHeader label="Resource Pages">
+                <CreateButton as={RouterLink} to={`${url}/createPage`}>
+                  Create page
+                </CreateButton>
+              </SectionHeader>
+              <SectionCaption icon={BiBulb} label="NOTE: ">
+                Pages are automatically ordered by latest date.
+              </SectionCaption>
+            </Box>
+          )}
```

Comment:
should we account for the fact that `isPagesEmpty` might change state due to loading? a probable sequence of events might be `true` (as before first fetch it defaults to `undefined`) -> successful load -> `false`, leading to this to be shown.

a quick fix would just be to have a skeleton wrap this (or to subsume it into the skeleton below)